warp10/src/main/java/io/warp10/script/ext/stackps/WSINFO.java

```diff
@@ -28,17 +29,9 @@
 import io.warp10.script.WarpScriptStackRegistry;
 
 public class WSINFO extends NamedWarpScriptFunction implements WarpScriptStackFunction {
-  
-  public static final String[] EXPOSED_ATTRIBUTES = new String[] {
-      WarpScriptStack.ATTRIBUTE_CREATION_TIME,
-      WarpScriptStack.ATTRIBUTE_FETCH_COUNT,
-      WarpScriptStack.ATTRIBUTE_GTS_COUNT,
-      WarpScriptStack.ATTRIBUTE_MACRO_NAME,
-      WarpScriptStack.ATTRIBUTE_NAME,
-      WarpScriptStack.ATTRIBUTE_SECTION_NAME,
-      StackPSWarpScriptExtension.ATTRIBUTE_SESSION,
-  };
-  
+
+  public static final String[] EXPOSED_ATTRIBUTES = new String[]{WarpScriptStack.ATTRIBUTE_CREATION_TIME, WarpScriptStack.ATTRIBUTE_FETCH_COUNT, WarpScriptStack.ATTRIBUTE_GTS_COUNT, WarpScriptStack.ATTRIBUTE_MACRO_NAME, WarpScriptStack.ATTRIBUTE_NAME, WarpScriptStack.ATTRIBUTE_SECTION_NAME, StackPSWarpScriptExtension.ATTRIBUTE_SESSION,};
```

Comment:
This is far harder to read than the original version, is that an IDE proposed change?

Reply:
Indeed, wrong auto formatting parameters. reverted